src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolder.java

```diff
@@ -386,7 +386,11 @@ public static <K, V extends TopLevelItem> Map<K, V> loadChildren(AbstractFolder<
                 configurations.put(key.apply(item), item);
             } catch (Exception e) {
                 File finalEffectiveSubdir = effectiveSubdir;
-                LOGGER.log(Level.WARNING, e, () -> "could not load " + finalEffectiveSubdir);
+                if (LOGGER.isLoggable(Level.FINE)) {
+                    LOGGER.log(Level.FINE, e, () -> "could not load " + finalEffectiveSubdir);
+                } else {
+                    LOGGER.log(Level.WARNING, "could not load " + finalEffectiveSubdir + " due to " + e);
+                }
```

Comment:
I would rather simplify

```suggestion
                LOGGER.warning(() -> "could not load " + finalEffectiveSubdir + " due to " + e);
                LOGGER.log(Level.FINE, null, e);
```